doc/source/cluster/index.rst

```diff
@@ -3,227 +3,33 @@
 
-        .. code-block:: bash
+Distributed Ray is powered by the Ray Cluster Launcher, which automatically provisions machines and launches a multi-node Ray cluster. You can use the cluster launcher on GCP, Amazon EC2, Azure, or even Kubernetes. Distributed Ray also makes use of the Ray Autoscaler, which gives Ray the ability "auto-scale," meaning that it can interact with a Cloud Provider to request or release instances according to application workload.
 
-            java -classpath <classpath> \
-              -Dray.address=<address> \
-              <classname> <args>
+How does it work?
+-----------------
 
-    .. note:: Specifying ``auto`` as the address hasn't been implemented in Java yet. You need to provide the actual address. You can find the address of the server from the output of the ``ray up`` command.
+The Ray Cluster Launcher will automatically enable a load-based autoscaler. The autoscaler resource demand scheduler will look at the task, actor, and placement group resource demands from the cluster, and try to add the minimum list of nodes that can fulfill these demands. When nodes are idle for more than :ref:`idle_timeout_minutes <cluster-configuration-idle-timeout-minutes>`, they will be removed. The head node is never removed.
 
+Autoscaler uses a simple binpacking algorithm to binpack the user demands into the available cluster resources. The remaining unfulfilled demands are placed on the smallest list of nodes the satisfies the demand while maximizing utilization (starting from the smallest node).
 
-.. note:: A common mistake is setting the address to be a cluster node while running the script on your laptop. This will not work because the script needs to be started/executed on one of the Ray nodes.
+**Here is "A Glimpse into the Ray Autoscaler" and how to debug/monitor your cluster:**
 
-To verify that the correct number of nodes have joined the cluster, you can run the following.
+2021-19-01 by Ameer Haj-Ali, Anyscale, Inc.
 
-.. code-block:: python
+.. youtube:: BJ06eJasdu4
 
-  import time
+Supported features
+------------------
 
-  @ray.remote
-  def f():
-      time.sleep(0.01)
-      return ray.services.get_node_ip_address()
+Distributed Ray can deliver support for a broad set of requirements. Among others it incorporates the following features:
 
-  # Get a list of the IP addresses of the nodes that have joined the cluster.
-  set(ray.get([f.remote() for _ in range(1000)]))
+* **Multiple node type autoscaling**: Ray supports the use of :ref:`multiple node types <cluster-configuration-available-node-types>` in a single cluster. In this mode of operation, the scheduler will choose the types of nodes to add based on the resource demands, instead of always adding the same kind of node type.
+* **Placement groups**: :ref:`Placement groups <ray-placement-group-doc-ref>` allow users to atomically reserve groups of resources across multiple nodes (i.e., gang scheduling).
```

Comment:
I know we call this a feature during development, but is this really a feature? I feel like it's table stakes, and it's implied that it exists.

Reply:
This isn't really specific to Ray clusters too right? Maybe this belongs more so in a usage guide.

Reply:
I'll delete the section since it's a bit random